terms/neuro/study.json

```diff
@@ -305,7 +305,7 @@
         },
         {
             "const": "JAX_APPPS1",
-            "description": "The JAX_APPPS1study - A mouse model of early onset AD",
+            "description": "The JAX_APPPS1study - A mouse model of early onset AD -- now refered to as the Jax.IU.Pitt_APP.PS1 study",
```

Comment:
Is the old annotation still in use?

Reply:
I just checked and I don't see either of the old study names in use. My understanding was that the old values were being updated to refer to the new study name mostly for record keeping. I would not be against removing the old values altogether, but would need to double check with Mette.

Reply:
Checked with Mette and she is fine with removing the old values.